lib/npg_qc/Schema/ResultSet.pm

```diff
@@ -10,14 +10,20 @@ extends 'DBIx::Class::ResultSet';
 our $VERSION = '0';
 
 sub search_autoqc {
-  my ($self, $values, $size) = @_;
+  my ($self, $query, $size) = @_;
 
   my $how = {'cache' => 1};
+  # Remap the query so that we do not change the input data,
```

Comment:
Clearly it's a good thing to do. However, it triggers an error in line 213 lib/npg_qc/mqc/outcomes.pm, which implicitly depends on undefined tag index being set to -1 for lib results in the previous line of code. So either we continue being sloppy by changing caller's data or fix the code for creating a new entry in the lib outcomes table.